Environment/Environment.pillar

```diff
@@ -1,3 +1,4 @@
+%TODO : A better par about the finder at the end of the chapter.
```

Comment:
Ok, i'll do it. I'm still pretty new to github so i didn't knew you could do that!